README.md

```diff
@@ -1,96 +1,75 @@
- - Download a test instance of VS Code
- - Download the appropriate version of ChromeDriver
- - Package and install your extension into the VS Code instance
- - Launch the VS Code instance using webdriver
- - Run your tests
-
-The second part is to provide an extendable page object API for more convenient test writing.
-
-## Usage
-
-Simply install it into your extension devDependencies to get started:
+<h1 align="center">
+	<br>
+	Visual Studio Code Extension Tester
+</h1>
+
+<p align="center">
+	<a href="https://www.npmjs.com/package/vscode-extension-tester"><img src="https://img.shields.io/npm/v/vscode-extension-tester?label=extester&color=orange&style=for-the-badge" alt="VSCode ExTester"/></a>
+    <a href="https://www.npmjs.com/package/vscode-extension-tester-locators"><img src="https://img.shields.io/npm/v/vscode-extension-tester-locators?color=orange&label=locators&style=for-the-badge" alt="ExTester Locators"/></a>
+    <a href="https://www.npmjs.com/package/monaco-page-objects"><img src="https://img.shields.io/npm/v/monaco-page-objects?color=orange&label=page%20objects&style=for-the-badge" alt="ExTester Page Objects"/></a>
+    <a href="https://github.com/redhat-developer/vscode-extension-tester/blob/main/LICENSE"><img src="https://img.shields.io/badge/license-Apache%202-blue.svg?style=for-the-badge&logo=appveyor" alt="License"/></a>
+    </br>
+    <a href="https://github.com/redhat-developer/vscode-extension-tester/actions/workflows/main.yml"><img src="https://img.shields.io/github/actions/workflow/status/redhat-developer/vscode-extension-tester/main.yml?label=Main%20CI&style=for-the-badge" alt="Main CI"></a>
+    <a href="https://github.com/redhat-developer/vscode-extension-tester/actions/workflows/insiders.yml"><img src="https://img.shields.io/github/actions/workflow/status/redhat-developer/vscode-extension-tester/insiders.yml?branch=main&label=Insider%20CI&style=for-the-badge" alt="Insider CI"></a>
+</p><br/>
+
+<h2 align="center">UI Testing Framework for Visual Studio Code.</h2>
+
+<p align="center">
+  <a href="#features">Features</a> •
+  <a href="#requirements">Requirements</a> •
+  <a href="#usage">Usage</a> •
+  <a href="https://github.com/redhat-developer/vscode-extension-tester/wiki/">Documentation</a> •
+  <a href="#issues">Issues</a> •
+  <a href="#faq">FAQ</a>
+</p>
+
+<p align="center">
+Extension Tester is a package that is designed to help you run UI tests for your <a href="https://code.visualstudio.com/">Visual Studio Code</a> extensions using <a href="https://www.npmjs.com/package/selenium-webdriver">selenium-webdriver</a>.
+</p><br/>
+
+### Features
+- Download a test instance of Visual Studio Code
+- Download the appropriate version of ChromeDriver
+- Package and Install your extension into downloaded VS Code instance
+- Launch the VS Code instance using Selenium Webdriver
+- Run your tests
+
+### Requirements
+ - **Visual Studio Code 1.37+**
+ - **NodeJS 16**
+ - **JDK 11+**
```

Comment:
Java?

Reply:
ops.. not strictly required.. I will remove that one.. nice catch! 👍